src/test/java/org/checkerframework/specimin/MethodReturnFullyQualifiedGenericTest.java

```diff
@@ -0,0 +1,20 @@
+package org.checkerframework.specimin;
+
+import java.io.IOException;
+import org.junit.Test;
+
+/**
+ * This test checks that Specimin includes the fully qualified class name in generics when
+ * generating a synthetic method. For example, if a return type is Bar<com.foo.Foo>, the actual
+ * synthetic return type should be com.qualified.Bar<com.foo.Foo>. At this moment, we should expect
+ * this test case to fail.
```

Comment:
Rather than "we should expect this test to fail", I would phrase this as "this test encodes Specimin's current behavior, which doesn't produce compilable output. If you break this test, it might not be a bad thing."